test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/DetailControlModelTests.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Xunit;
+using Moq;
+using NuGet.ProjectManagement;
+using NuGet.Test.Utility;
+using Microsoft.VisualStudio.Threading;
+using NuGet.VisualStudio;
+using System.Threading;
+using System;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.UI.Test.Models
+{
+    public abstract class DetailControlModelTestBase : IClassFixture<LocalPackageSearchMetadataFixture>, IDisposable
+    {
+        protected readonly LocalPackageSearchMetadataFixture _testData;
+        protected readonly PackageItemListViewModel _testViewModel;
+        protected readonly JoinableTaskContext _joinableTaskContext;
+        protected bool disposedValue = false;
+
+        public DetailControlModelTestBase(LocalPackageSearchMetadataFixture testData)
+        {
+            _testData = testData;
+            _testViewModel = new PackageItemListViewModel()
+            {
+                LocalPackageInfo = _testData.TestData.LocalPackageInfo,
+                Version = new NuGetVersion(0, 0, 1),
```

Comment:
Do these need to be two distinct NuGetVersion objects?

Reply:
Good catch! Only one version is needed